pkg/processor/sumologicsyslogprocessor/processor_test.go

```diff
@@ -38,16 +38,15 @@ func TestProcessLogs(t *testing.T) {
 	}
 
 	logs := pdata.NewLogs()
-	logs.ResourceLogs().Resize(1)
-	logs.ResourceLogs().At(0).InstrumentationLibraryLogs().Resize(len(lines))
-	lgs := logs.ResourceLogs().At(0).InstrumentationLibraryLogs().At(0).Logs()
+	rls := logs.ResourceLogs().AppendEmpty()
+	rls.InstrumentationLibraryLogs().EnsureCapacity(len(lines))
+	lgs := rls.InstrumentationLibraryLogs().AppendEmpty()
 
 	for _, line := range lines {
-		lr := pdata.NewLogRecord()
+		lr := lgs.Logs().AppendEmpty()
 		lr.Body().SetStringVal(line)
-		lgs.Append(lr)
 	}
-	lgs.At(1).Attributes().InsertString("facility_name", "pre filled facility")
+	lgs.Logs().At(1).Attributes().InsertString("facility_name", "pre filled facility")
```

Comment:
`lgs` can be misleading now

Reply:
Granted that it's `InstrumentationLibraryLogs` I could call it `ils`, WDYT?

Reply:
Done